### ISSUE TITLE
[macOS] Specify window can not be maximized after being moved

### ISSUE THREAD

**Issue by maxxxxxdlp** [specify/specify6#571]
TL;DR: After being maximized and moved, Specify can not be maximized again

If the main Specify window is maximized (green button), macOS sees it as being a regular window stretched to occupy the entire screen. Normally, apps would transition to the fullscreen mode

Since Specify implements a maximized window in a weird way, the user is able to move the window while it is maximized, by clicking on the top bar (the one that occupies window control buttons and `Specify 6.8.00` label) and moving it. At this point, the window is the size of the full screen but moved to a different position, so it no longer occupies the entire screen

Normally, in this case the user can click on the maximize button again and the program will return to being maximized and covering the entire screen. Yet, at this point, that button has no effect in Specify. User needs to manually resize a window to any different size for that button to start working again

![Screen Shot 2020-01-28 at 3 00 56 PM](https://user-images.githubusercontent.com/40512816/73305109-168be200-41df-11ea-9640-aa1fb1a90ff3.png)

Tested in macOS 10.15

**Comment by maxxxxxdlp** [comment]
Reproduced in macOS 10.13 and 10.14
